test/unit/.eslintrc

```diff
@@ -5,5 +5,8 @@
   "globals": {
     "expect": true,
     "sinon": true
+  },
+  "rules": {
+    "no-unused-expressions": "off"
```

Comment:
Why needs this rule to be disabled? backwards compatibility to the old version?

Reply:
Hi Christian,
thanks for the review.

You're right, there's a compability issue for the unit tests, in particular chai assertions, e.g.
```
expect(vm.tileGridDefs).to.be.empty;
```
leads to an _non-unused-expression_ warning.
You can read more about it here (https://stackoverflow.com/questions/37558795/nice-way-to-get-rid-of-no-unused-expressions-linter-error-with-chai).

Note that I disabled this rule within the scope of unit tests only, the source code of Wegue itself is not affected.

Reply:
Thanks for clarification. Makes totally sense. Ready to merge now...